lib/shared.bash

```diff
@@ -173,7 +173,12 @@ function build_image_override_file_with_version() {
 
 # Runs the docker-compose command, scoped to the project, with the given arguments
 function run_docker_compose() {
-  local command=(docker-compose)
+  local command
+  if [[ "$(plugin_read_config CLI_VERSION "1")" == "1" ]] ; then
+    command=(docker-compose)
+  else
+    command=(docker compose)
+  fi
```

Comment:
I believe it would be simpler (and more in line with my other comment to do this instead):

```suggestion
  local command=(docker-compose)
  if [[ "$(plugin_read_config CLI_VERSION "1")" == "2" ]] ; then
    command=(docker compose)
  fi
```